lib/ansible/modules/cloud/azure/azure_rm_postgresqlserver.py

```diff
@@ -71,7 +71,30 @@
     create_mode:
         description:
             - Create mode of SQL Server
-        default: Default
+        default: default
+        choices:
+            - default
+            - geo_restore
+            - point_in_time_restore
+    source_server_id:
```

Comment:
does it take dict of resource  group and name? if not, better to  support it.

Reply:
not yet, I can do it to follow standard